test/integration/provider/rti.go

```diff
@@ -39,7 +37,8 @@ func (r *ResourcesTrackerImpl) InitializeResourcesTracker(machineClass *v1alpha1
 
 	if err == nil {
 		r.InitialInstances = instances
-		volumes, err := getAvailableVolumes(ctx, svc, clusterTag, clusterTagValue, machineClass, secretData)
+		//since disk name is same as instance name
```

Comment:
Are you sure the disk names are same as instances? I mean, no suffixes like `-disk` or `-os-disk`?

Reply:
Yes I have tried it by creating orphan disks, you can also verify on the GCP console:)